viz/charts/linkerd-viz/README.md

```diff
@@ -99,6 +99,7 @@ Kubernetes: `>=1.13.0-0`
 | grafana.resources.memory.request | string | `nil` | Amount of memory that the grafana container requests |
 | identityTrustDomain | string | `"cluster.local"` | Trust domain used for identity  |
 | imagePullSecrets | list | `[]` | For Private docker registries, authentication is needed.  Registry secrets are applied to the respective service accounts |
+| installNamespace | bool | `true` | Create the namespace or not |
```

Comment:
Small nit but can we use wording similar to the [base chart](https://github.com/linkerd/linkerd2/blob/2087c95dd8cec0b11e6ab04d1abf4a4019410977/charts/linkerd2/README.md#L200)?

```suggestion
| installNamespace | bool | `true` | Set to false when installing in a custom namespace. |
```

Reply:
Very small nit, They can still set this true  even for custom namespace usecase too right? where they would set `namespace: xyz`

Reply:
Right,  the message can be a little misleading. Something that can be addressed in the upcoming Helm values cleanup work :wink:

Reply:
Ah right sorry, that's my fault. Sounds good about fixing it.